src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -189,6 +189,7 @@ public void subscribeToCertificateUpdates(GetCertificateRequest getCertificateRe
             }
             metrics.subscribeSuccess();
         } catch (NoSuchAlgorithmException e) {
+            metrics.subscribeError();
```

Comment:
might be splitting hairs, but it may be good to move usage of the new ClientDeviceAuthMetrics methods to a separate PR.